sycl/doc/design/CompilerAndRuntimeDesign.md

```diff
@@ -434,7 +434,7 @@ It returns 0 if no kernels are present and 1 otherwise.
 
 #### Device code post-link step
 
-At link time all the device code is always linked into a single LLVM IR module.
+At link time all the device code is linked into a single LLVM IR module unless `-fno-sycl-rdc` is specified.
```

Comment:
Please, keep 80-character line length limit.

Reply:
sorry i had too many expectations for clang format :)